cocktail/src/cocktail.js

```diff
@@ -0,0 +1,27 @@
+import {React} from "react";
+import { Link } from 'react-router-dom'
+
+const Cocktail = (props) => {
+    const { image, name, id, info, glass } = props
+    return(
+        // <div className="cocktail">
+        //     <p>hi</p>
+        // </div>
+      <article className='cocktail'>
+      <div className='img-container'>
+        <img src={image} alt={name} />
+      </div>
+      <div className='cocktail-footer'>
+        <h3>{name}</h3>
+        <h4>{glass}</h4>
+        <p>{info}</p>
+        <Link to={`/cocktail/${id}`} className='btn btn-primary btn-details'>
+          details
+        </Link>
+      </div>
+    </article>
+    )
+}
+
+export default Cocktail;
```

Comment:
It will be worth to understand the difference between named export and default export.
Also, why they should be used and when?